### PR TITLE
remark-npm2yarn: support npx command

### PR DESCRIPTION
**Pull request by peterkogo**
`npm-to-yarn` supports conversions of `npx` commands since [version 3.0.0](https://github.com/nebrelbug/npm-to-yarn/releases/tag/v3.0.0).

Currently, this is prevented by the hard-coded error thrown from the library. I extended the condition to also accept commands starting with `npx`.

Haven't added a changeset because I am not familiar with your guidelines.

### CODE REVIEW

**Review by dimaMachina — CHANGES_REQUESTED** [review]
build fails

File: packages/remark-npm2yarn/src/plugin.ts [hunk 1]
```diff
@@ -97,9 +97,9 @@ export const remarkNpm2Yarn: Plugin<
 
       if (!node.meta || node.meta === newMetadata) return;
 
-      if (!node.value.startsWith('npm')) {
+      if (!node.value.startsWith('npm') || !node.value.startsWith('npx')) {
```
Comment (dimaMachina) [inline packages/remark-npm2yarn/src/plugin.ts:100]: this line has a problem

Comment (peterkogo) [reply packages/remark-npm2yarn/src/plugin.ts:100]: Omg, that's supposed to be a `&&` there.

Comment (dimaMachina) [reply packages/remark-npm2yarn/src/plugin.ts]: no worries this happens with everybody 😅

**Review by dimaMachina — APPROVED** [review]

**Pull request merged**